Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1

```diff
@@ -427,6 +511,14 @@ function Set-ClusterProtectionLeveltoEncryptandSignforServiceFabric
             catch
             {
                 $ServiceFabricSkipped += $ServiceFabric
+                if($AutoRemediation){	
+                $logResource = @{}	
+                $logResource.Add("ResourceGroupName",($_.ResourceGroupName))	
+                $logResource.Add("ResourceName",($_.ResourceName))	
+                $logResource.Add("Reason","Encountered error while fetching app service configuration")    	
```

Comment:
Edit the reason : App Service is mentioned here

Reply:
this is done, thanks